### PR TITLE
My part of translation has been completed. It's so happy!

### PR DESCRIPTION
**Pull request by Xueliang24**

### CODE REVIEW

**Comment by MichaelChirico** [comment]
Thanks @Xueliang24! The encoding issue surfaced again; have resolved on web UI.

Could you please review @propran?

File: po/R-zh_CN.po [hunk 1]
```diff
@@ -1342,18 +1352,22 @@ msgid ""
...
+“因此显示的值可能（使用户）留下‘值是不会重叠，但其内部表现显示显示却重叠’的印象，”
+“（所以）请确保POSIXct类型的间隔列具有相同的‘时区’属性以避免混乱。”
 
 msgid "Not yet implemented"
-msgstr ""
+msgstr "尚未实现"
 
 msgid "maxgap > minoverlap. maxgap will have no effect here."
-msgstr ""
+msgstr "最大空隙大于最小重叠，此处的最大空隙则无效。"
```
Comment (Emberwhirl) [inline po/R-zh_CN.po]: 最大空隙大于最小重叠，此处的最大空隙无效

**Comment by MichaelChirico** [comment]
@Emberwhirl please let me know if your review is complete :)

File: po/zh_CN.po [hunk 2]
```diff
@@ -2589,114 +2589,125 @@ msgstr ""
...
 msgid ""
 "Invalid value integer64='%s'. Must be 'integer64', 'character', 'double' or "
 "'numeric'"
 msgstr ""
+"64位整数型有效值='%s'.必须是'64位整数型'，'字符串'，'双精度浮点型'或者'数值型'"
 
 #: freadR.c:176
 msgid "Use either select= or drop= but not both."
-msgstr ""
+msgstr "要么使用select=要么使用drop=，禁止同时使用"
```
Comment (Emberwhirl) [inline po/zh_CN.po]: line 2613：select=和drop=不可同时使用

File: po/R-zh_CN.po [hunk 3]
```diff
@@ -1267,73 +1267,83 @@ msgid "A non-empty vector of column names or numbers is required for by.x"
...
 
 msgid "Elements listed in 'by.x' must be valid names in data.table 'x'"
-msgstr ""
+msgstr "对于data.table中的‘X’，‘by.x’中的元素必须是有效名称"
 
 msgid ""
 "Duplicate columns are not allowed in overlap joins. This may change in the "
 "future."
 msgstr ""
+“搭接部分不允许出现重复列。”
```
Comment (MichaelChirico) [inline po/R-zh_CN.po]: i think Github is yelling about the quotes not being standard `"`

**Comment by MichaelChirico** [comment]
Thanks so much @Xueliang24 and @Emberwhirl !!!

There were some updates, mainly due to using the wrong quote marks (麻烦, i know), you can see here

https://github.com/Rdatatable/data.table/pull/3935/commits/cf9298413a9a0208144cc03725bcd3e2e0159d8f

**Comment by Xueliang24** [comment]
Thanks so much your reminding and reviewing again!!! I will notice the
correct style of quote marks😁

Michael Chirico <notifications@github.com> 于2019年12月19日周四 下午12:47写道：

**Comment by Emberwhirl** [comment]
Thanks for the reminder. I will pay attention to the quote marks in future.

**Pull request merged**